src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -220,6 +237,8 @@ const MenuItem: VibeComponent<MenuItemProps> & {
       [setSubMenuIsOpenByIndex, index, closeMenu]
     );
 
+    const iconLabel = typeof title === "string" ? title : ariaLabel;
+
     const renderSubMenuIconIfNeeded = () => {
       if (!hasChildren) return null;
```

Comment:
I would say not to take title's type into account. aria || title

Reply:
it fails TS, it's not smart enough